neard/Cargo.toml

```diff
@@ -1,7 +1,15 @@
+[[bin]]
+path = "src/main.rs"
+name = "neard"
+
+[[bin]]
+path = "src/../src/main.rs"
```

Comment:
why?

Reply:
That's on master for a while like that:
https://github.com/nearprotocol/nearcore/blob/master/neard/Cargo.toml#L58